# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../libcap/libcap-1.10-ia64.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2004 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Pulled from RedHat SRPM to (mostly) fix samba (3.0.x) compilation on
non-x86 systems. Most of this patches are bare hacks - but seem to be
commonly accepted ... :-(

  - Rene Rebe <rene.rebe@gmx.net>

--- libcap-1.10/libcap/cap_sys.c.ia64	Mon May 21 16:23:27 2001
+++ libcap-1.10/libcap/cap_sys.c	Mon May 21 16:24:09 2001
@@ -11,14 +11,6 @@
 #define __LIBRARY__
 #include <linux/unistd.h>
 
-_syscall2(int, capget,
-	  cap_user_header_t, header,
-	  cap_user_data_t, data)
-
-_syscall2(int, capset,
-	  cap_user_header_t, header,
-	  const cap_user_data_t, data)
-
 /*
  * $Log: cap_sys.c,v $
  * Revision 1.1.1.1  1999/04/17 22:16:31  morgan
--- libcap-1.10/Make.Rules.ia64	Mon May 21 16:22:08 2001
+++ libcap-1.10/Make.Rules	Mon May 21 16:22:32 2001
@@ -44,10 +44,10 @@
 CC=gcc
 COPTFLAGS=-O2
 DEBUG=-g #-DDEBUG
-WARNINGS=-ansi -D_POSIX_SOURCE -Wall -Wwrite-strings \
+WARNINGS=-D_POSIX_SOURCE -Wall -Wwrite-strings \
         -Wpointer-arith -Wcast-qual -Wcast-align \
         -Wtraditional -Wstrict-prototypes -Wmissing-prototypes \
-        -Wnested-externs -Winline -Wshadow -pedantic
+        -Wnested-externs -Winline -Wshadow
 LD=ld
 LDFLAGS=-s #-g
 
